pilot/pkg/networking/util/util.go

```diff
@@ -280,6 +280,14 @@ func ConvertLocality(locality string) *core.Locality {
 	}
 }
 
+// isEmpty checks if a locality is empty (checking region is good enough, based on how its initialized)
```

Comment:
Either add a comment that begins with the function public name (lint error) or make it a private function